files/tree_manipulation.go

```diff
@@ -18,11 +18,11 @@ func SortDesc(folder *File) {
 	}
 }
 
-func pruneFolder(folder *File, limit int64) {
+func PruneSmallFiles(folder *File, limit int64) {
```

Comment:
This is the only change in the code apart from renaming. I had to export this function because it's used in the new `commands` package.